analyzer/mfcckeeper.cc

```diff
@@ -27,6 +27,7 @@
 
 #undef max
 #undef min
+// Who thought these macros were a good idea, again?
```

Comment:
:)

Probably best not to antagonize whoever added the macros, and additionally to add a bit more clarity, in case the reader doesn't understand the issue with the macros. How about something more like:

// Explicitly undef these to avoid conflicts with their std:: equivalents.

Also as a small note, The comment should be moved to line 27 so that the comment explains what's about to come rather than explaining after the fact.